h2o-py/h2o/estimators/infogram.py

```diff
@@ -0,0 +1,969 @@
+     as models built using all attributes.
+    """
+
+    algo = "infogram"
+    supervised_learning = True
+
+    def __init__(self,
+                 model_id=None,  # type: Optional[Union[None, str, H2OEstimator]]
+                 training_frame=None,  # type: Optional[Union[None, str, H2OFrame]]
+                 validation_frame=None,  # type: Optional[Union[None, str, H2OFrame]]
+                 seed=-1,  # type: int
+                 keep_cross_validation_models=True,  # type: bool
+                 keep_cross_validation_predictions=False,  # type: bool
+                 keep_cross_validation_fold_assignment=False,  # type: bool
+                 fold_assignment="auto",  # type: Literal["auto", "random", "modulo", "stratified"]
+                 fold_column=None,  # type: Optional[str]
+                 response_column=None,  # type: Optional[str]
+                 ignored_columns=None,  # type: Optional[List[str]]
+                 ignore_const_cols=True,  # type: bool
+                 score_each_iteration=False,  # type: bool
+                 offset_column=None,  # type: Optional[str]
+                 weights_column=None,  # type: Optional[str]
+                 standardize=False,  # type: bool
+                 distribution="auto",  # type: Literal["auto", "bernoulli", "multinomial", "gaussian", "poisson", "gamma", "tweedie", "laplace", "quantile", "huber"]
+                 plug_values=None,  # type: Optional[Union[None, str, H2OFrame]]
+                 max_iterations=0,  # type: int
+                 stopping_rounds=0,  # type: int
+                 stopping_metric="auto",  # type: Literal["auto", "deviance", "logloss", "mse", "rmse", "mae", "rmsle", "auc", "aucpr", "lift_top_group", "misclassification", "mean_per_class_error", "custom", "custom_increasing"]
+                 stopping_tolerance=0.001,  # type: float
+                 balance_classes=False,  # type: bool
+                 class_sampling_factors=None,  # type: Optional[List[float]]
+                 max_after_balance_size=5.0,  # type: float
+                 max_confusion_matrix_size=20,  # type: int
+                 max_runtime_secs=0.0,  # type: float
+                 custom_metric_func=None,  # type: Optional[str]
+                 auc_type="auto",  # type: Literal["auto", "none", "macro_ovr", "weighted_ovr", "macro_ovo", "weighted_ovo"]
+                 infogram_algorithm="gbm",  # type: Literal["auto", "deeplearning", "drf", "gbm", "glm", "xgboost"]
+                 infogram_algorithm_params=None,  # type: Optional[dict]
+                 protected_columns=None,  # type: Optional[List[str]]
+                 net_information_threshold=4.940656458e-324,  # type: float
+                 total_information_threshold=4.940656458e-324,  # type: float
+                 safety_index_threshold=4.940656458e-324,  # type: float
+                 relevance_index_threshold=4.940656458e-324,  # type: float
```

Comment:
that's pretty ugly: one more reason to use `-1` to represent `AUTO`.

Reply:
agreed and changed.